WordPress/src/main/java/org/wordpress/android/ui/accounts/login/MagicLinkRequestFragment.java

```diff
@@ -150,9 +155,8 @@ public void onErrorResponse(VolleyError error) {
                 HashMap<String, String> errorProperties = new HashMap<>();
                 errorProperties.put(ERROR_KEY, error.getMessage());
                 AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_MAGIC_LINK_FAILED, errorProperties);
-                if (getView() != null) {
-                    Snackbar.make(getView(), R.string.magic_link_unavailable_error_message, Snackbar.LENGTH_SHORT);
-                }
+                mProgressDialog.cancel();
+                ToastUtils.showToast(getActivity(), R.string.magic_link_unavailable_error_message, ToastUtils.Duration.LONG);
```

Comment:
There should be an `isAdded()` check before using `getActivity()` in case the fragment is no longer valid (the `getView() != null` check previously did this).